docs/ingestion/data-formats.md

```diff
@@ -1020,6 +1026,27 @@ For details, see the Schema Registry [documentation](http://docs.confluent.io/cu
 ...
 ```
 
+Multiple Instances:
+```json
+...
+"avroBytesDecoder" : {
+   "type"   : "schema_registry",
+   "urls"   : [<schema-registry-url-1>, <schema-registry-url-2>, ...],
+   "config" : {
+               "schema.registry.basic.auth.credentials.source" : "USER_INFO",
+               "schema.registry.basic.auth.user.info"          : "fred:letmein",
+               ... 
+              },
+   "headers": {
+               "traceID"   : "b29c5de2-0db4-490b-b421",
+               "timeStamp" : "1577191871865",
+               ...
+               
+             }
```

Comment:
Also since this config section contains potentially sensitive information, longer term it might make sense to use something like what is suggested in #10309 (though obviously not in this PR)